src/Uno.UI/UI/Xaml/Controls/Primitives/Thumb.cs

```diff
@@ -127,7 +144,8 @@ protected override void OnPointerCaptureLost(PointerRoutedEventArgs args)
 			base.OnPointerCaptureLost(args);
 
 			// Note: We don't handle event as UWP does not, and otherwise it will prevent parent control to update its visual state
-			CompleteDrag(args.GetCurrentPoint(this).Position);
+			// Note2: Position MUST be absolute as the element might have moved
+			CompleteDrag(args.GetCurrentPoint(null).Position);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'args' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5062186)